TingClientAgencyBranch.php

```diff
@@ -287,6 +287,20 @@ public function getIsOclcRsLibrary() {
     return isset($this->pickupAgency->isOclcRsLibrary) ? $this->pickupAgency->isOclcRsLibrary->{'$'} : NULL;
   }
 
+  /** Check if branch will receive ILL requests
+   * @return bool
+   */
+  public function getWillReceiveIll() {
+    return (isset($this->pickupAgency->willReceiveIll) && $this->pickupAgency->willReceiveIll->{'$'} == '1') ? TRUE : FALSE;
+  }
+
+  /** Get text describing why branch will not receive ILL requests
+   * @return string
```

Comment:
@return kan også være NULL - Dette er dog flueknep :-D

Reply:
Måske skulle det være en tom streng i stedet for null?

Reply:
Den er lavet på samme måde som andre funktioner i dette modul, så jeg bør faktisk rette dem alle til - men så skal vi også have fat i alt der bruger dette modul, og sikre os at der ikke længere testes for NULL... Dokumentationen kan jeg dog godt rette :)